aten/src/ATen/native/vulkan/ops/Packing.cpp

```diff
@@ -0,0 +1,160 @@
+#include <ATen/native/vulkan/ops/Packing.h>
+
+namespace at {
+namespace native {
+namespace vulkan {
+namespace ops {
+
+using namespace api::utils;
+
+vTensor pack_image2d_h2w2(vTensor v_src, api::Context* context, api::Command::Buffer& command_buffer) {
```

Comment:
const correctness please.  Pass vTensor by const ref, and change `api::Context* context` to `api::Context* const context` to mark the pointer (not the pointee) immutable.